.github/workflows/radar-release-actions.yml

```diff
@@ -25,4 +26,4 @@ jobs:
           ORG_GRADLE_PROJECT_SIGNINGKEY: ${{ secrets.ORG_GRADLE_PROJECT_SIGNINGKEY }}
           ORG_GRADLE_PROJECT_SIGNINGPASSWORD: ${{ secrets.ORG_GRADLE_PROJECT_SIGNINGPASSWORD }}
           SNAPSHOT: false
-        run: ./gradlew build publish
```

Comment:
we will likely want to pull the file from master for both edited files in `.github/workflows`
```
git checkout origin/master -- path/to/file
```